### DISCUSSION TITLE
Roundcube is not returning records values

### DISCUSSION THREAD

**Discussion by yeti92force** [YetiForceCompany/YetiForceCRM#16814] [How to]
Hi dear support,

Roundcube mail composer is not returning fields values when i compose my mail.
![image](https://user-images.githubusercontent.com/107400809/212728415-d2d01be5-1374-495e-befc-0fea1d2a8921.png)

I'm using yetiforce 5.3

However, on a yetiforce 6.4, all is working.
![image](https://user-images.githubusercontent.com/107400809/212728639-c77ed3fc-387f-4f23-b277-5eea9130ddd9.png)

Any idead ? I'm trying to compose a mail from task widget

Regards

**Comment by mariuszkrzaczkowski** [comment]
Maybe in 5.3 there was no such functionality, update to 6.4
